Sources/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -745,6 +747,37 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
     }
 }
 
+//MARK: Universal Bridge support
+extension MiniAppScriptMessageHandler {
+    func sendJsonToHostApp(requestParam: RequestParameters?, callbackId: String) {
+        guard let requestParamValue = requestParam?.jsonInfo else {
+            executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: MiniAppJavaScriptError.invalidPermissionType.rawValue)
```

Comment:
```suggestion
            executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: prepareMAJavascriptError(MiniAppJavaScriptError.unexpectedMessageFormat)
```

Reply:
Changes are updated in the latest commits.